edge/samples_utilities/webutils/adminutils/README.md

```diff
@@ -0,0 +1,68 @@
+# MVI Edge Adminutils
+
+This web sample demonstrates the use of the Controller REST API for a number of functions that are not exposed in the UI.
+
+The `Login/Logout` button will create and end sessions with the controller.
```

Comment:
I am confused by what the intent of this file is.
It is difficult for me to tell what is "normal" UI and what is "extra" stuff.
Login/Logout seems normal, UI Notification popups described later is "normal" UI operation.
Why wouldn't this information be in the pubs?

Reply:
🤣 As you noted in your main comment on the PR, this is all "extra" stuff.